proto/grpc/proxy/v2/proxy.proto

```diff
@@ -106,4 +111,10 @@ service LoadTestService {
             get: "/v2/load-test"
         };
     }
+
+    rpc Delete (DeleteRequest) returns (google.protobuf.Empty) {
```

Comment:
IMO it makes sense to use `DeleteResponse` type as result with empty fields list to make it extensible in case we'l need it.

Reply:
Done via 116dbce.